include/deal.II/grid/grid_tools.h

```diff
@@ -247,6 +247,29 @@ namespace GridTools
   double
   cell_measure(const T &, ...);
 
+  /**
+   * This function computes an affine approximation of the map from the unit
+   * coordinates to the real coordinates of the form $p_\text{real} = A
+   * p_\text{unit} + b $ by a least squares fit of this affine function to the
+   * $2^\text{structdim}$ vertices representing a quadrilateral or hexahedral
+   * cell. For any valid mesh cell whose geometry is not degenerate, this
+   * operation results in a unique affine mapping. Thus, this function will
+   * return a finite result for all given input points, even in cases where
+   * the actual transformation by an actual bi-/trilinear or higher order
+   * mapping might be singular. The result is exact in case the transformation
+   * from the unit to the real cell is indeed affine, such as in one dimension
+   * or for Cartesian and affine (parallelogram) meshes in 2D/3D.
+
+   * This approximation is underlying the function
+   * TriaAccessor::real_to_unit_cell_affine_approximation() function.
+   *
+   * For exact transformations to the unit cell, use
+   * Mapping::transform_real_to_unit_cell().
+   */
+  template <int structdim, int spacedim>
+  std::pair<DerivativeForm<1, structdim, spacedim>, Tensor<1, spacedim>>
```

Comment:
What is the first and the second component of the pair?

Reply:
I will fix that in #10992.